### ISSUE TITLE
permitted_attributes `

### ISSUE THREAD

**Issue by hjhart** [Dreamersoul/administrate-field-active_storage#143]
When I have a project using 
    administrate (0.19.0)
    rails (7.0.7.2)
    administrate-field-active_storage (1.0.1)

And a model that looks like this:

```
# == Schema Information
#
# Table name: reviews
#
#  id            :bigint           not null, primary key
#  text          :string
#  restaurant_id :bigint           not null
#  created_at    :datetime         not null
#  updated_at    :datetime         not null
#
class Review < ApplicationRecord
  belongs_to :restaurant
  has_many_attached :photos
end
```

And active storage set up, I get this error when trying to create or update a review with `photos` provided.

ArgumentError::ReviewsController#update

wrong number of arguments (given 1, expected 0)
```
# permitted for has_many_attached   
def permitted_attributes     
  super + [photos: []]  
end
```

I followed the documentation closely, so I'm not sure what I'm doing wrong. I also tried the above `permitted_attributes` file in the `reviews_controller.rb` instead of the `reviews_dashboard.rb` file, but hit a different error. 

Is there something obvious I'm doing wrong?

Thanks!

**Comment by hjhart** [comment]
Figured this out. Maybe it's a newer version of `administrate` potentially. 

But this fixes it:

```
  # permitted for has_many_attached
  def permitted_attributes(_action = nil)
    super + [photos: []]
  end
```

Instead of 
```
  # permitted for has_many_attached
  def permitted_attributes
    super + [photos: []]
  end
```

This code here shows that's the proper signature now: https://github.com/thoughtbot/administrate/blob/2d667c1cb514774f90cbcfbed82273f32c7f406b/lib/administrate/base_dashboard.rb#L71-L85

**Comment by hjhart** [comment]
Yeah, 9 months ago they changed that method signature with this commit: 
https://github.com/thoughtbot/administrate/commit/fd61e4d818daefcbb7c9e088ac42f6d5b3e51e9c

Could be worth updating the documentation?

**Comment by Dreamersoul** [comment]
Yes I think it is, can you do it and I'll merge it 👍🏻
